src/Sql/AbstractSql.php

```diff
@@ -291,6 +305,58 @@ protected function processSubSelect(
         return $decorator->buildSqlString($platform, $driver, $parameterContainer);
     }
 
+    public function processJoin(
```

Comment:
Should be protected (per the other methods that are peers to it). I'll make that change on merge.